content/en/docs/devops-user-guide/how-to-use/use-pipeline-templates.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Use Pipeline Templates"
+keywords: 'KubeSphere, Kubernetes, Jenkins, Graphical Pipelines, Pipeline Templates'
+description: 'Understand how to use pipeline templates on KubeSphere.'
+linkTitle: "Use Pipeline Templates"
+weight: 11290
+---
+
+KubeSphere offers a graphical editing panel where the stages and steps of a Jenkins pipeline can be defined through interactive operations. In KubeSphere v3.1, two built-in pipeline templates are provided as frameworks of continuous integration (CI) and continuous delivery (CD).
+
+When you have a pipeline created in your DevOps project on KubeSphere, you can click the pipeline to go to its detail page, and then click **Edit Pipeline** to select a pipeline template based on your needs. This document illustrates the concept of these two pipeline templates.
+
+## CI Pipeline Template
+
+![ci-template](/images/docs/devops-user-guide/using-devops/use-pipeline-templates/ci-template.png)
+
+![ci-stages](/images/docs/devops-user-guide/using-devops/use-pipeline-templates/ci-stages.png)
+
+The CI pipeline template contains two stages. The **clone code** stage checks out code and the **build & push** stage builds an image and pushes it to Docker Hub. You need to create the credentials for your code repository and your Docker Hub registry in advance, and then set the URL of your repository and these credentials in corresponding steps. After you finish editing, the pipeline is ready to run.
+
+## CI & CD Pipeline Template
+
+![cicd-template](/images/docs/devops-user-guide/using-devops/use-pipeline-templates/cicd-template.png)
+
+![cicd-stages](/images/docs/devops-user-guide/using-devops/use-pipeline-templates/cicd-stages.png)
+
+The CI & CD pipeline template contains six stages. For more information about what each stage does, refer to [Create a Pipeline Using a Jenkinsfile](../create-a-pipeline-using-jenkinsfile/#pipeline-overview) where you can find similar stages and the descriptions. You need to create a kubeconfig credential and the credentials for your code repository and your Docker Hub registry in advance, and then set the URL of your repository and these credentials in corresponding steps. After you finish editing, the pipeline is ready to run.
```

Comment:
For more information about what each stage does
->
For more information about each stage

You need to create a kubeconfig credential and the credentials for your code repository and your Docker Hub registry in advance
->
You need to create a kubeconfig credential, the credentials for your code repository, and your Docker Hub registry in advance

Reply:
If that's the case, just confirm two things with you:
1. Do we need to change the title of this doc to Credentials Management;
![Xnip2021-04-28_18-36-11](https://user-images.githubusercontent.com/75110798/116391252-7e513280-a851-11eb-999c-81cd88611ae9.png)

2. how about change `a kubeconfig credential and the credentials for your code repository and your Docker Hub registry` to `credentials for your code repository, your Docker Hub registry, and the kubeconfig of your cluster`

Reply:
1. I think Credential Management and Credentials Management are both ok to me. In most cases, we do not need to use the plural form of the first word if two nouns are combined as a phrase unless it is really necessary. I think we can keep Credential Management in this case.
2. Yes, that sounds better.